pkg/datastore/eventstore.go

```diff
@@ -90,6 +90,7 @@ func (s *eventStore) UpdateEventStatus(ctx context.Context, eventID string, stat
 		event.StatusDescription = statusDescription
 		if event.IsHandled() {
 			event.HandledAt = s.nowFunc().Unix()
+			event.UpdatedAt = s.nowFunc().Unix()
```

Comment:
nit: just to ensure they are using the same value

```suggestion
			now = s.nowFunc().Unix()
			event.HandledAt = now
			event.UpdatedAt = now
```

Reply:
Good catch!